src/coreclr/vm/yieldprocessornormalized.cpp

```diff
@@ -108,6 +110,14 @@ static double MeasureNsPerYield(unsigned int measureDurationUs)
     return Max(MinNsPerYield, Min((double)elapsedTicks * NsPerS / ((double)yieldCount * ticksPerS), MaxNsPerYield));
 }
 
+void YieldProcessorNormalization::GCHeapSetYieldProcessorScalingFactor()
+{
+    if (GCHeapUtilities::IsGCHeapInitialized())
```

Comment:
Maybe this could be an assertion, as this function shouldn't be called until it has already been initialized

Reply:
I was thinking about an assertion too but I thought it could be possible that the first scheduled measurement happens before the GC heap is initialized. Although that won't happen right?